ui-admin/src/study/participants/participantList/ParticipantListViewSwitcher.tsx

```diff
@@ -1,52 +1,41 @@
-import { Button } from 'components/forms/Button'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faUserLarge, faUserLargeSlash } from '@fortawesome/free-solid-svg-icons'
+import { faList, faUserLarge, faUserLargeSlash } from '@fortawesome/free-solid-svg-icons'
 import { faUsers } from '@fortawesome/free-solid-svg-icons/faUsers'
 import React from 'react'
+import { NavLink } from 'react-router-dom'
+import { StudyEnvironmentConfig } from '@juniper/ui-core'
 
-type ParticipantView = 'participant' | 'family' | 'withdrawn'
+const VIEWS = [{
+  name: 'participant',
+  icon: faUserLarge,
+  path: ''
+},  {
+  name: 'account',
+  icon: faList,
+  path: '/accounts'
+}, {
+  name: 'family',
+  icon: faUsers,
+  path: '/families'
+}, {
+  name: 'withdrawn',
+  icon: faUserLargeSlash,
+  path: '/withdrawn'
+}]
 
-type ParticipantListViewSwitcherProps = {
-    view: ParticipantView
-    setView: (view: ParticipantView) => void
-    setSearchParams: (params: { view: string }) => void
-    familyLinkageEnabled: boolean
-}
-
-export const ParticipantListViewSwitcher = (props: ParticipantListViewSwitcherProps) => {
-  const { view, setView, setSearchParams, familyLinkageEnabled } = props
+export const ParticipantListViewSwitcher = ({ studyEnvConfig }: {studyEnvConfig: StudyEnvironmentConfig}) => {
+  const availableViews = VIEWS.filter(view => view.name !== 'family' || studyEnvConfig.enableFamilyLinkage)
   return (
     <div className="btn-group border my-1">
-      <Button variant='light'
-        aria-label={'Switch to withdrawn view'}
-        className={`btn btn-sm ${view === 'withdrawn' ? 'btn-dark' : 'btn-light'}`}
-        tooltip={'Switch to withdrawn view'}
```

Comment:
it looks like we lost the tooltips here (although there's still alt text on hover). i was going to suggest that we keep the tooltips, but that made me realize that the importance of keeping the tooltips probably means the icons aren't nearly intuitive enough 🙂 

i don't think there's a great icon to distinguish this new view either. stylistically i like `faUsersBetweenLines` or `faUsersGear` but i don't think they do a great job of conveying their purpose

so +1 in favor of switching this to a dropdown view switcher. doesn't have to be a part of this PR though.

Reply:
100% agree, I think a dropdown would be preferable here. I'm sure this is a somewhat solved UX problem, too, I think it would be worth spending some time to find other examples of "table view switching" UX options since the participant list is so central to our app.